server/bin/pbench-verify-backup-tarballs.py

```diff
@@ -0,0 +1,349 @@
+#!/usr/bin/env python3
+# -*- mode: python -*-
+
+import os
+import sys
+import glob
+import hashlib
+import shutil
+import tempfile
+from enum import Enum
+from argparse import ArgumentParser
+from s3backup import S3Config, Entry
+from pbench import init_report_template, report_status, _rename_tb_link, \
+    PbenchConfig, BadConfig, get_es, get_pbench_logger, md5sum
+
+_NAME_    = "pbench-verify-backup-tarballs"
+
+class Status(Enum):
+    SUCCESS = 10
+    FAIL = 20
+
+class BackupObject(object):
+    def __init__(self, name, dirname):
+        self.name = name
+        self.dirname = dirname
+        self.list_name = "list.{}".format(self.name)
+        self.description = self.name
+
+
+def sanity_check(s3_obj, logger):
+    # make sure the S3 bucket exists
+    try:
+        s3_obj.connector.head_bucket(Bucket='{}'.format(s3_obj.bucket_name))
+    except Exception:
+        logger.exception(
+            "Bucket: {} does not exist or you have no access\n".format(s3_obj.bucket_name))
+        s3_obj = None
+
+    return s3_obj
+
+
+def checkmd5(target_dir, tmpdir, backup_obj, logger):
```

Comment:
Add an explanatory comment aboutr what checkmd5()( returns and what the return value means (sys.maxsize  meaning failure is not obvious).